configure

```diff
@@ -3452,6 +3453,19 @@ else
 
 fi
 
+# Check whether --disable-gcc-info was given.
+if test "${disable_debug_info+set}" = set; then :
+  enableval=$disable_debug_info;
+fi
+
+if test "x$disable_debug_info" != x; then :
+  disable_debug_info=""
+
+else
```

Comment:
Modify `configure.ac` and run `autoconf` rather than modify configure script directly.